rules/manager.go

```diff
@@ -287,6 +287,12 @@ func (g *Group) run(ctx context.Context) {
 		return
 	}
 
+	ctx = promql.NewOriginContext(ctx, map[string]string{
+		"groupFile":          g.File(),
+		"groupName":          g.Name(),
+		"evaluationInterval": g.Interval().String(),
```

Comment:
We don't need this, it's implicit from the file/group. Plus it'll be obvious from the timestamps.